features/apimgt/org.wso2.carbon.apimgt.rest.api.store.feature/src/main/resources/api-docs/docs/models/ApplicationKeyGenerateRequest.html

```diff
@@ -109,6 +109,36 @@ <h2>ApplicationKeyGenerateRequest</h2>
                         <td class="parameter"> <p class="marked">null</p></td>
                         <td class="parameter"> <p class="marked">[&quot;am_application_scope&quot;,&quot;default&quot;]</p></td>
                     </tr>
+                    <tr>
+                        <td class="parameter">
+                            <p>clientId</p>
+                            <p class="param-required">
+                                
+                                optional
+                            </p>
+                        </td>
+                        <td class="parameter"><p class="marked">Client ID for generating access token.</p></td>
+                        <td class="parameter">
+                                <a href="#!/models#string">string</a>
+                        </td>
+                        <td class="parameter"> <p class="marked">null</p></td>
+                        <td class="parameter"> <p class="marked"></p></td>
+                    </tr>
+                    <tr>
+                        <td class="parameter">
+                            <p>clientSecret</p>
+                            <p class="param-required">
+                                
```

Comment:
Same here.

Reply:
These are generated code. Do we need to modify these.

Reply:
Then it would be fine.